packages/frontend/src/system-integration/webxdc.ts

```diff
@@ -44,10 +56,10 @@ export async function internalOpenWebxdc(
   const chatName = (
     await BackendRemote.rpc.getBasicChatInfo(accountId, message.chatId)
   ).name
-  const displayname = await BackendRemote.rpc.getConfig(
-    accountId,
-    'displayname'
-  )
+  const account: Type.Account =
+    await BackendRemote.rpc.getAccountInfo(accountId)
```

Comment:
It would be nice to only do the backend call only if the `displayname` is empty.

Reply:
But that would need 2 calls, the getConfig call and a second if displayName is empty

Reply:
Ohh, yes, you're right. I missed the fact that `getConfig` is async.